lib/roger_ui/jobs.ex

```diff
@@ -1,6 +1,53 @@
-defmodule RogerUi.Jobs do
+defmodule RogerUI.Jobs do
   @moduledoc """
-  Generate Jobs list from Roger.Info.running_jobs()
+  Generates a Jobs list from Roger.Info.running_jobs() function.
+  Resulting Jobs are linked to a partition name:
+
+  Given a nested data structure, where each element contents nested items:
+  [
```

Comment:
Can you put code examples in markdown code blocks?

Reply:
done